WebDriverAgentTests/UnitTests/FBSDKVersionTests.m

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "FBRuntimeUtils.h"
+
+@interface FBSDKVersionTests : XCTestCase
+@property (nonatomic, readonly) NSString *currentSDKVersion;
+@property (nonatomic, readonly) NSString *lowerSDKVersion;
+@property (nonatomic, readonly) NSString *higherSDKVersion;
+@end
+
+@implementation FBSDKVersionTests
+
+- (void)setUp
+{
+  [super setUp];
+  NSDictionary *bundleDict = [[NSBundle mainBundle] infoDictionary];
+  [bundleDict setValue:@"11.0" forKey:@"DTSDKName"];
```

Comment:
Hacky like hell :).
Easier approach might be having two layer API, one where you pass current version explicitly (that one you test) and second convenience one that passes current SDK version.

Tho we can merge this one and fix it if there is a problem.